site/_posts/2020-11-06-custom-admin-registration-forms.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: blog-post
+title: Update user profile data in the admin UI
```

Comment:
How to update user profile data in the FusionAuth admin UI

Reply:
Or "Using advanced registration forms to update user profile data"

Reply:
Hmm, you're right that it doesn't pop.

What we are basically doing is letting FusionAuth admin users who build custom user profile editing forms.

How about 

* Build custom user profile edit forms using FusionAuth
* How to build custom profile edit forms within FusionAuth
* Manage custom user profile data in the FusionAuth admin